Core/functions.php

```diff
@@ -27,7 +27,7 @@ function env(): Env
 {
     $env = Env::get();
 
-    if ($env === null) {
+    if (!isset($env)) {
         throw new LogicException('Env is not set');
     }
```

Comment:
wouldn't comparing to null be faster cause there's no function call overhead, no?

Reply:
Yet again, nanoseconds faster. Also `isset` is not an actual function, it's a language construct, so it gets a different approach from the PHP interpreter and, iirc, it compiles into a different op code. Ultimately, using `isset()` is practically the same as strict comparison to `null`, but IMO the first one improves readability.